From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Thu, 20 Aug 2026 20:59:52 +0200
Subject: userVerifier: Report init error for all InitError failures

Getting the user verifier proxy may report init errors, and we should at
least notify the user about instead of leaving them with an empty screen
---
 js/gdm/userVerifier.js | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/js/gdm/userVerifier.js b/js/gdm/userVerifier.js
index 353e834..f19b4ce 100644
--- a/js/gdm/userVerifier.js
+++ b/js/gdm/userVerifier.js
@@ -116,6 +116,10 @@ export class ShellUserVerifier extends Signals.EventEmitter {
             if (!anyBegan && lastInitError)
                 this._reportInitError(lastInitError);
         } catch (e) {
+            if (e instanceof InitError) {
+                this._reportInitError(e);
+                return;
+            }
             logErrorUnlessCancelled(e);
         }
 
